edge-lambda/src/wellcomeLibraryEncoreRedirect.ts

```diff
@@ -140,12 +140,17 @@ export const requestHandler = async (
   // only have '/search' as the path and then put the search terms in the query string.
   const searchPathRegExp: RegExp = /^\/iii\/encore\/search.*/;
 
+  // URLs like https://search.wellcomelibrary.org/iii/encore/?lang=eng
+  const encoreHomepage = '/iii/encore/';
+
   if (path.match(bibPathRegExp)) {
     return getWorksRedirect(path);
   } else if (path.match(accountPathRegExp)) {
     return wellcomeCollectionRedirect('/account');
   } else if (path.match(searchPathRegExp)) {
     return getSearchRedirect(path, qs);
+  } else if (path === encoreHomepage) {
+    return wellcomeCollectionRedirect('/collections/');
```

Comment:
We were already redirecting `/iii/encore/` to the collections homepage (which is correct), but because it wasn't matching we were getting a spurious warning in the CloudWatch Logs:

> Unable to redirect /iii/encore/